examples/icon.js

```diff
@@ -23,6 +23,7 @@ var iconStyle = new ol.style.Style({
     anchor: [0.5, 46],
     anchorXUnits: 'fraction',
     anchorYUnits: 'pixels',
+    color: '#8959A8',
```

Comment:
I think it would be nicer to have a dedicated example that shows what this option should be used for.  We have a bad tendency to continue modifying examples that started out simple until they become overly complex.  And I think this particular example looked nicer before the `color` option was added (same goes for `opacity`).

![image](https://cloud.githubusercontent.com/assets/41094/12224960/9eeff544-b7bb-11e5-9560-5dce128dde70.png)